cmd/start.go

```diff
@@ -46,17 +48,19 @@ func init() {
 	startCmd.Flags().String("crd-version", "v1", "the version of the CRD you want monitored")
 	startCmd.Flags().String("crd-namespace", metav1.NamespaceNone, "(optional) the namespace of the CRD you want monitored, only needed for namespaced CRDs (ex: default)")
 	startCmd.Flags().String("templates", "", "absolute path to the directory with your template files")
-	startCmd.Flags().String("metrics-address", ":8080", "The address and port for the metrics endpoint")
+	startCmd.Flags().String("default-address", ":8080", "The address and port for endpoints such as /metrics and /status")
 	startCmd.Flags().String("metrics-endpoint", "/metrics", "The URI for the metrics endpoint")
+	startCmd.Flags().String("status-endpoint", "/status", "The URI for the status endpoint")
 
 	viperBindFlag("k8s.config", startCmd.Flags().Lookup("kube-config"))
 	viperBindFlag("crd.name", startCmd.Flags().Lookup("crd-name"))
 	viperBindFlag("crd.group", startCmd.Flags().Lookup("crd-group"))
 	viperBindFlag("crd.version", startCmd.Flags().Lookup("crd-version"))
 	viperBindFlag("crd.namespace", startCmd.Flags().Lookup("crd-namespace"))
 	viperBindFlag("templates", startCmd.Flags().Lookup("templates"))
-	viperBindFlag("metrics.address", startCmd.Flags().Lookup("metrics-address"))
+	viperBindFlag("default.address", startCmd.Flags().Lookup("default-address"))
```

Comment:
Can we change this. Maybe `listener.address`, `listener.metrics_uri` and `listener.status_uri`

I'd be okay with `server` instead of `listener` but these should all be nested under the same section.